modules/admin/client/components/search.api.js

```diff
@@ -0,0 +1,10 @@
+export async function searchUsers(query) {
```

Comment:
@Akronix FYI since you wanted to use the same API elsewhere. Perhaps this fetch can be added in a separate PR before both of you need it?

Reply:
It could be the same API endpoint and just accept more query parameters for users with admin role and return more data with that role, too.

That said, might be more secure to keep things like that behind `/api/admin*`

Reply:
At any rate, Instead of `/components/search.api.js` (because it's not a React component) perhaps `/api` or `/services`?

Reply:
When I move it to services I get
```
/Users/guaka/trustroots/modules/admin/client/services/search.api.js
  1:1  error  Parsing error: The keyword 'export' is reserved
```

cc @mrkvon

Reply:
@guaka add `modules/admin/client/services/search.api.js` to overrides[1].files (the overrides for react) in `.eslintrc.js`. For now, we need to specify, which code is migrated and which not.

Reply:
> Instead of /components/search.api.js (because it's not a React component) perhaps /api or /services